development/github/pkg/repoowners/owners.go

```diff
@@ -0,0 +1,163 @@
+package repoowners
```

Comment:
Isn't this package somehow available from the `k8s.io/test-infra`? Do we really need to implement it?